### DISCUSSION TITLE
Hour format

### DISCUSSION THREAD

**Discussion by Potopi** [mathoudebine/turing-smart-screen-python#433] [Q&A] [answered]
Hi, is there a way to force an hour format like XX:XX:XX? Currently, when it gets past midnight, it shows X:XX:XXY, where Y is the last digit of the previous hour (a 9) and it just stays there until 10am. Any way to force it like 00:01:38?

**Comment by Peeecha** [comment, marked as answer]
Classic case of ghost character that is left on the screen, happens when the new string is shorter than it was before.
A simple hacky tweak would be to format the time to fill the width to max length.
Changing this line:
https://github.com/mathoudebine/turing-smart-screen-python/blob/58b9a49a156087351ba10dc47445e1b01c0e7d99/library/stats.py#L496

to:
`value=f"{babel.dates.format_time(date_now, format=time_format, locale=lc_time):>8}"`

This would work if you are using medium format (XX:XX:XX), which looks like you are. You can also left align it with **:<8**, if thats preferred.

Another option, if you are happy with the heading 0, is to just update your theme by adding **FORMAT: hh:mm:ss** under your hour stats settings.

**Reply by Potopi** [reply]
Thank you very much, I just changed windows hour format, as you sugested, and it's working now as I intended. Cheers mate.
